<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AWGBCheckToolClass</class>
 <widget class="QMainWindow" name="AWGBCheckToolClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1194</width>
    <height>812</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1194</width>
    <height>812</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AWGBCheckTool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="30,70">
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>ËµÑÊ∫êÊ†ë</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QTreeView" name="treeView"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,5">
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>ÂΩìÂâçÈÄöÈÅì</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="5,5,0">
          <property name="spacing">
           <number>7</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>ÂΩìÂâçÊ£ÄÊµãÈ°πÔºö</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>ÔºàÊó†Ôºâ</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="verticalSpacing">
           <number>7</number>
          </property>
          <item row="0" column="1">
           <widget class="QPushButton" name="pushButton_5">
            <property name="text">
             <string>ÁõÆÂΩïÊü•ËØ¢</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButton_6">
            <property name="text">
             <string>‰∏ä‰∏ÄÈ°π</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButton_7">
            <property name="text">
             <string>‰∏ã‰∏ÄÈ°π</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButton_8">
            <property name="text">
             <string>Ê≥®ÂÜå</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item alignment="Qt::AlignLeft">
           <widget class="QPushButton" name="pushButton_9">
            <property name="text">
             <string>‰ø°‰ª§</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QStackedWidget" name="stackedWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="page">
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>20</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>‰ø°‰ª§ÊòæÁ§∫Âå∫Ôºö</string>
                  </property>
                 </widget>
                </item>
                <item alignment="Qt::AlignRight">
                 <widget class="QPushButton" name="pushButton_11">
                  <property name="text">
                   <string>Ê∏ÖÈô§</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QTextBrowser" name="textBrowser"/>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Ê£ÄÊµãÁªìÊûúÊòæÁ§∫Âå∫Ôºö</string>
                  </property>
                 </widget>
                </item>
                <item alignment="Qt::AlignRight">
                 <widget class="QPushButton" name="pushButton_12">
                  <property name="text">
                   <string>Ê∏ÖÈô§</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QTableView" name="tableView"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_2">
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="8,2">
              <property name="rightMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QOpenGLWidget" name="openGLWidget"/>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,3,2,2,2,1,3,1,3,1,3,1">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <spacer name="verticalSpacer_4">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_13">
                  <item>
                   <widget class="QPushButton" name="pushButton_24">
                    <property name="text">
                     <string>Êí≠Êîæ</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_25">
                    <property name="text">
                     <string>ÂÅúÊ≠¢</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_17">
                  <item alignment="Qt::AlignHCenter">
                   <widget class="QPushButton" name="pushButton_32">
                    <property name="text">
                     <string>üëÜ</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_18">
                  <item>
                   <widget class="QPushButton" name="pushButton_33">
                    <property name="text">
                     <string>üëà</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_34">
                    <property name="text">
                     <string>üëâ</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_19">
                  <item alignment="Qt::AlignHCenter">
                   <widget class="QPushButton" name="pushButton_35">
                    <property name="text">
                     <string>üëá</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <spacer name="verticalSpacer_5">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_14">
                  <item>
                   <widget class="QPushButton" name="pushButton_26">
                    <property name="text">
                     <string>Áº©Â∞è</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_27">
                    <property name="text">
                     <string>ÊîæÂ§ß</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <spacer name="verticalSpacer_6">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_15">
                  <item>
                   <widget class="QPushButton" name="pushButton_28">
                    <property name="text">
                     <string>ÊãâËøë</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_29">
                    <property name="text">
                     <string>ÊãâËøú</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <spacer name="verticalSpacer_7">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_16">
                  <item>
                   <widget class="QPushButton" name="pushButton_30">
                    <property name="text">
                     <string>ÂΩïÂÉè</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_31">
                    <property name="text">
                     <string>ÂõûÊîæ</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <spacer name="verticalSpacer_9">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QStackedWidget" name="stackedWidget_2">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="page_3">
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,0,5,0,5,0">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>20</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item alignment="Qt::AlignRight">
                 <widget class="QPushButton" name="pushButton_13">
                  <property name="text">
                   <string>ËΩ¨Âà∞ËÆæÂ§áÂàóË°®</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="5,1,5,1,5">
                <item>
                 <widget class="QPushButton" name="pushButton_14">
                  <property name="text">
                   <string>ÊéßÂà∂1</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_15">
                  <property name="text">
                   <string>ÊéßÂà∂2</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_16">
                  <property name="text">
                   <string>ÊéßÂà∂3</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="5,1,5,1,5">
                <item>
                 <widget class="QPushButton" name="pushButton_17">
                  <property name="text">
                   <string>ÊéßÂà∂4</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_4">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_18">
                  <property name="text">
                   <string>ÊéßÂà∂5</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_5">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_19">
                  <property name="text">
                   <string>ÊéßÂà∂6</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_4">
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,0">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="0,5">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>ËÆæÂ§áÂàóË°®Ôºö</string>
                  </property>
                 </widget>
                </item>
                <item alignment="Qt::AlignRight">
                 <widget class="QPushButton" name="pushButton_23">
                  <property name="text">
                   <string>ËΩ¨Âà∞ÊéßÂà∂</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QTableView" name="tableView_2"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_5">
           <widget class="QLabel" name="label_7">
            <property name="geometry">
             <rect>
              <x>290</x>
              <y>40</y>
              <width>221</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>ËßÜÈ¢ë‰ø°ÊÅØ: PS,RTP ÔºàÂæÖÂÆöÔºâ</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_8">
            <property name="geometry">
             <rect>
              <x>290</x>
              <y>90</y>
              <width>191</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>ÁêÜËß£PSÂ∞ÅË£ÖÂíåRTPÂ∞ÅË£ÖËßÑÂàôÂêé</string>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
